modules/virt-configuring-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-obsolete-cpu-models_{context}"]
+= Configuring obsolete CPU models
+
+Configure a list of obsolete CPU models by editing the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+* Edit the `HyperConverged` custom resource, specifying the obsolete CPU models in the `obsoleteCPUs` array. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  obsoleteCPUs:
+    cpuModels: <1>
+      - "<obsolete_cpu_1>"
+      - "<obsolete_cpu_2>"
+    minCPUModel: "<minimum_cpu_model>" <2>
+----
+<1> Replace the example values in the `cpuModels` array with obsolete CPU models. Any value that you specify is added to a predefined list of obsolete CPU models. The predefined list is not visible in the CR.
+<2> Replace this value with the minimum CPU model that you want to use for basic CPU features. If you do not specify a different value, `Penryn` is used as the `minCPUModel`, though it is not visible in the CR by default.
```

Comment:
"If you do not specify a different value"
Do you mean "different value" or a "value"? Are you saying "If you do not change the value" or "If you do not specify a value"? 

Suggestion: Replace this value with the minimum CPU model that you want to use for basic CPU features. If the value is not specified, Penryn is used as the default.

Not sure you need to mention that it is not visible in CR by default because the user can see that it is not there. They just need to know what happens if they do leave the parameter empty.